src/pages/index.tsx

```diff
@@ -1,21 +1,184 @@
-/* eslint-disable no-unused-vars */
 import * as React from "react";
-import { Link } from "gatsby";
-
 import Layout from "../components/layout";
-import Image from "../components/image";
-import SEO from "../components/seo";
+import { Table as AntTable, Typography } from "antd";
+import { ColumnsType } from "antd/es/table";
```

Comment:
I added some further configuration to Typescript, rebasing to main should fix this.